packages/react-storage/src/components/StorageBrowser/displayText/libraries/en/uploadView.ts

```diff
@@ -79,6 +80,19 @@ export const DEFAULT_UPLOAD_VIEW_DISPLAY_TEXT: DefaultUploadViewDisplayText = {
 
     return { content: 'All files uploaded.', type };
   },
+  getFilesValidationMessage: ({ invalidFiles } = {}) => {
+    if (!invalidFiles) {
+      return undefined;
+    }
+    const fileNames = invalidFiles
+      .filter(({ file }) => file.size > UPLOAD_FILE_SIZE_LIMIT)
+      .map(({ file }) => file.name)
+      .join(', ');
+    return {
+      content: `File ${fileNames} exceeds the size limit 160 GB. Failed to add to upload queue.`,
```

Comment:
If I understood correctly the 160GB limit is per file? Can the message be something like "These files cannot be added to the upload queue due to they are larger than 160GB respectively" could be clearer?